cvat/apps/engine/permissions.py

```diff
@@ -399,6 +434,34 @@ def create_scope_create(cls, request: ExtendedRequest, org_id: int | None):
             org_role=getattr(membership, 'role', None),
             scope=cls.Scopes.CREATE)
 
+    @classmethod
+    def get_scope_specific_params(
+        cls,
+        scope: Scopes,
+        *,
+        request: ExtendedRequest,
+        view: ViewSet,
+        obj: Project | None,
+        iam_context: IamContext | None
+    ):
+        params = {}
+
+        if DownloadExportedExtension.Scopes.DOWNLOAD_EXPORTED_FILE == scope:
+            DownloadExportedExtension.extend_params_with_rq_job_details(
+                request=request, params=params
+            )
+
+        if scope in (
+            cls.Scopes.EXPORT_ANNOTATIONS,
+            cls.Scopes.EXPORT_BACKUP,
+            cls.Scopes.EXPORT_DATASET,
+        ) and obj:
+            ExportableResourceExtension.update_scope_params.__func__(
```

Comment:
Why not just `cls.update_scope_params(...)`?

Reply:
Why not give them unique method names, then?

Reply:
Actually, both these 2 extensions already could use the same names for both implemented methods. There's an option to resolve it by adding some prefix or suffix to the function name.